src/Symfony/Component/AssetMapper/Tests/Compiler/JavaScriptImportPathCompilerTest.php

```diff
@@ -93,6 +93,16 @@ public function testCompileFindsCorrectImports(string $input, array $expectedJav
 
     public static function provideCompileTests(): iterable
     {
+        yield 'import_in_multiline_comment' => [
+            'input' => <<<EOF
+/**
+ *     <LazyMotion features={() => import('./path/to/domAnimations')} />
+ * ```
+ */
+EOF,
+            'expectedJavaScriptImports' => [],
+        ];
+
```

Comment:
This test is already passing on 6.4 and 7.1 (on my machine).. Could you double-check ?

Reply:
Then perhaps the test is wrong.  The underlying problem is with an API Platform js file, causing this trivial example to fail.

```bash
symfony new MyApp  --webapp && cd MyApp
composer req api
bin/console make:entity -a Task -n
symfony server:start -d
symfony open:local --path=/api
```

Reply:
I'm just trying to get a "before" / "after" here ;)  Without it i cannot be sure if what i'm doing has any effect :)